nodejs/week1/src/backend/routes/cheap-meals.js

```diff
@@ -0,0 +1,32 @@
+//Respond with the json for all the meals
+//(including it's reviews) that are cheap (you define what a cheap meal is)
+
+const express = require("express");
+const app = express();
+
+const mealsData = require("../data/meals.json");
+// get reviews from the reviews.json
+const reviewData = require("../data/reviews.json");
+
+// if the price of meal is below 85 then it is cheap meal
+const cheapMeal = mealsData.filter((meal) => meal.price < 75);
+
+//compare two json files to add reviews that matches a meal id
+cheapMeal.forEach((meal) => {
+  //iterate over meals data
+  const matchedReviewData = []; // a temporary array to hold review data when there is a match
+  reviewData.forEach((review) => {
+    //iterate over reviews data
+    if (meal.id == review.meal_id) {
+      //if the meal has a review
+      matchedReviewData.push(review);
+    }
+    meal["reviews"] = matchedReviewData;
```

Comment:
This works, but you should move this line 2 lines below, outside of the `reviewData.forEach()` loop so you only assign `matchedReviewData` to `meal.reviews` once for each meal (it's more efficient).

Same for all the other files where you use the same approach. 👍

Reply:
implemented this